.github/workflows/desktop.yml

```
@@ -227,14 +227,7 @@ jobs:
           name: build-logs-${{ matrix.triplet }}
           path: |
             ${{ env.CMAKE_BUILD_DIR }}/vcpkg-manifest-install.log
-
-      - name: 📑 Upload Dep Build Logs
-        uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: build-logs-${{ matrix.triplet }}
-          path: |
-            ${{ env.CMAKE_BUILD_DIR }}/vcpkg-manifest-install.log
+            /Users/runner/src/buildtrees/**/*.log
```

That looks 🐠

---

Yeah, that was just for me to get osx logs. I don't know what the proper fix is but the env based path is wrong, we get 0 logs.

---

${VCPKG_ROOT}/buildtrees/**/*.log

?

---

@m-kuhn , nope, try this and logs are missing (see https://github.com/opengisch/QField/actions/runs/2178613351)

---

```suggestion
            ${{ steps.vars.outputs.VCPKG_ROOT }}/src/buildtrees/**/*.log
```

That should work better